src/frontend/Sider/Behandling/Vilkårvurdering/validering.ts

```diff
@@ -1,42 +1,68 @@
 import { BegrunnelseRegel, Regel, RegelId, Regler } from '../../../typer/regel';
+import { erDatoEtterEllerLik } from '../../../utils/dato';
+import { harIkkeVerdi } from '../../../utils/utils';
 import { Delvilkår } from '../vilkår';
 
-export type Feilmeldinger = Record<RegelId, string | undefined>;
+export type Feilmeldinger = {
+    delvilkårsvurderinger: Record<RegelId, string | undefined>;
+    fom?: string;
+    tom?: string;
+};
+
+export const ingenFeil = { delvilkårsvurderinger: {} };
+
+export function ingen(valideringsfeil: Feilmeldinger) {
```

Comment:
`ingen` ble litt vanskelig å lese følte jeg, `utenFeilmeldinger` kanskje? 
```suggestion
export ingen = (valideringsfeil: Feilmeldinger) => {
```

Reply:
Hehe ja, er enig, jeg likte bare at bruken av den ble 

`if (ingen(valideringsfeil)) { ... }`

Reply:
Skal uansett gå litt gjennom dette når jeg flytter det ut i egen hook